charts/rancher-external-ip-webhook/rancher-external-ip-webhook/100.0.1+up1.0.1/Chart.yaml

```diff
@@ -7,7 +7,7 @@ annotations:
   catalog.cattle.io/release-name: rancher-external-ip-webhook
   catalog.cattle.io/ui-component: rancher-external-ip-webhook
```

Comment:
needs rancher-version and upstream-version Chart annotations, will also ned to update the index.yaml for the new annotations after that.

Reply:
done